src/ripple/rpc/handlers/Connect.cpp

```diff
@@ -52,7 +53,7 @@ Json::Value doConnect (RPC::JsonContext& context)
     if(context.params.isMember (jss::port))
         iPort = context.params[jss::port].asInt ();
     else
-        iPort = 6561;
+        iPort = DEFAULT_PEER_PORT;
```

Comment:
This is going to require a documentation change. https://xrpl.org/connect.html
Not a deal breaker, but important to bring to @mDuo13's attention. We also need to consider if this is going to cause backward-compatibility issues. I doubt it will, because no nodes I know of run on that port.

I'll add the "Documentation" label now.